packages/ui/tsconfig.spec.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "outDir": "../../dist/out-tsc",
+        "module": "commonjs",
+        "types": ["jest", "node"]
+    },
+    "include": [
+        "**/*.test.ts",
+        "**/*.spec.ts",
+        "**/*.test.tsx",
+        "**/*.spec.tsx",
+        "**/*.test.js",
+        "**/*.spec.js",
+        "**/*.test.jsx",
+        "**/*.spec.jsx",
+        "**/*.d.ts"
```

Comment:
```suggestion
        "**/*.spec.ts"
```

Reply:
As far as I know this files are generated by using nx command. Is there any chance to update autogenerating process to avoid manual updates for each new package in the future?

Reply:
As far as I know this files are generated by using nx command. Is there any chance to update autogenerating process to avoid manual updates for each new package in the future?

Reply:
done

Reply:
@serhii-tkachenko  There is a ticket open to improve that generator; however, before that ticket is completed, I propose that we manually edit the package after using the generator or copying an existing package.

Reply:
@serhii-tkachenko  There is a ticket open to improve that generator. however, before that ticket is completed, I propose that we manually edit the package after using the generator or copying an existing package.

Reply:
@bc-peng Thanks for sharing 🙏